### PR TITLE
Test enhancement

### PR DESCRIPTION
**Pull request by peter279k**
# Changed log
- add more tests.
- integrate the Travis build. See Travis build [log](https://travis-ci.org/peter279k/uuidifier/builds/361680149).
- fix the Test name typo.

### CODE REVIEW

File: tests/UudifierTest.php [hunk 1]
```diff
@@ -1,88 +0,0 @@
-<?php
-
-use PHPUnit\Framework\TestCase;
-use Ramsey\Uuid\UuidInterface;
-use Teamleader\Uuidifier\Uuidifier;
-use Ramsey\Uuid\Uuid;
-
-class UudifierTest extends TestCase
```
Comment (stivni) [inline tests/UudifierTest.php]: why did you remove our testcases?

Comment (mrnnn) [reply tests/UudifierTest.php]: The file is gone though. Looks like you forgot to commit something.

Next to that, we don't have Travis in our setup. We may add some other CI though, but not right now.

**Comment by mrnnn** [comment]
Hi Peter. We won't go ahead and merge this PR as we don't want any of the travis stuff or config changes. Thanks for contributing though.
